sensor/kubernetes/listener/watcher/virtualmachine/availabilitychecker.go

```diff
@@ -0,0 +1,23 @@
+package virtualmachine
+
+import (
+	"github.com/stackrox/rox/pkg/k8sapi"
+	"github.com/stackrox/rox/pkg/virtualmachine"
+	"github.com/stackrox/rox/sensor/kubernetes/client"
+	"github.com/stackrox/rox/sensor/kubernetes/listener/watcher/availability"
+)
+
+type availabilityChecker interface {
```

Comment:
This is the same for all watchers, right? Could we move to a common interface like `availability.Checker`?